kythe/javatests/com/google/devtools/kythe/platform/kzip/KZipWriterTest.java

```diff
@@ -77,8 +77,12 @@ private void testWrite(KZip.Encoding encoding) throws IOException, KZipException
     // Read in the provided kzip.
     InMemoryKZip originalKZip = readKZip(TestDataUtil.getTestFile("stringset.kzip"));
 
+    // Delete old tmp kzip (if present).
+    File tmpKZipFile = new File(getTestTempDir(), "write_test.kzip");
+    tmpKZipFile.delete();
+
     // Write out our kzip.
-    KZipWriter writer = new KZipWriter(new File(getTestTempDir(), "write_test.kzip"), encoding);
```

Comment:
So I think the real issue is that the temp file is reused across tests. Rather than deleting it, let's make sure it has a unique name. Maybe just prepend encoding.toString() to the name?

Reply:
Sounds good. Done.